src/components/GenericPagesComponents/Select/Select.tsx

```diff
@@ -97,9 +97,13 @@ const SelectSpan = styled(OakSpan)`
   ${ellipsis}
 `;
 
-export function Select<T extends object>(
-  props: AriaSelectProps<T> & SelectProps,
-) {
+export function Select<
+  T extends {
+    // Set type of value as string, is this right?
```

Comment:
I think if it works and TS doesn't complain that it's likely right. Can remove this comment before merging.